build/rbac-setup.sh

```diff
@@ -66,7 +66,7 @@ export CYPRESS_BASE_URL=https://`oc get route multicloud-console -n $acm_install
 # test oauth server and see if idp has been setup
 i=0
 while true; do
-  IDP=`curl -sSL -k ${CYPRESS_BASE_URL} | grep ${OC_IDP}` || true
+  IDP=`curl -sSL -k "${CYPRESS_BASE_URL}/oauth/start?rd=/" | grep ${OC_IDP}` || true
```

Comment:
I'm wondering if we should even still have this check... since if it fails, the tests are skipped. Well, worse than skipped, they aren't even registered so it's hard to tell that something went wrong.

This check now depends on a specific url path, which certainly could change in the future.

Reply:
Fair point. I suppose we could start our Cypress tests with this check instead?

Reply:
Yeah - maybe we can short-circuit if the IDP isn't there, so the rest of the RBAC tests get skipped, but we still get one failure we can be alerted to? And the non-rbac tests would still get to run, I'm assuming.